apiserver/facades/client/bundle/bundle.go

```diff
@@ -150,11 +160,149 @@ func (b *BundleAPI) GetChanges(args params.BundleChangesParams) (params.BundleCh
 	return results, nil
 }
 
+var fail = func(failErr error) (params.StringResult, error) {
```

Comment:
Please move this declaration to inside the method where it is relevant. Only ExportBundle uses it.